modules/application/src/main/java/edu/northeastern/truthtree/controller/stories/Stories.java

```diff
@@ -0,0 +1,34 @@
+package edu.northeastern.truthtree.controller.stories;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import edu.northeastern.truthtree.dto.StoryDTO;
+import edu.northeastern.truthtree.service.stories.IStoriesService;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+public class Stories implements IStories {
+  private IStoriesService service;
+
+  @Autowired
+  public Stories(IStoriesService service) {
+    this.service = service;
+  }
+
+  @RequestMapping(value = "/api/story", method = RequestMethod.POST)
```

Comment:
Story creation should be at `/api/stories`

Reply:
There is just one story that will be created in this api so /api/story instead of /api/stories. Alternatively, it will be fetching list of stories so /api/stories for fetch call